nengo_dl/tests/test_converter.py

```diff
@@ -995,7 +996,8 @@ def test_time_distributed():
 
     _test_convert(inp, x, temporal_model=True)
 
-    with pytest.raises(
-        TypeError, match="can only be converted when temporal_model=True"
-    ):
-        converter.Converter(tf.keras.Model(inp, x), temporal_model=False)
+    if timesteps is not None:
```

Comment:
This error should be raised in both cases I think

Reply:
No, I think if `timesteps is None`, then it's the `ValueError` in `ConvertInput` that we were discussing above. That's why I added that comment about setting `temporal_model=True` there, since if `timesteps is None` and `temporal_model=False`, that's the first error the user sees. We could test that other error here, too, though.

Reply:
Ah yeah that's true I guess there isn't an obvious way to hit this in the `timesteps=None` case. I might catch the other error here, just to make it clear that it's an error in both cases (so no one reading this test thinks it should work in the other case, like my mistake).